common/src/main/java/org/jboss/pnc/buildagent/common/security/Checksum.java

```diff
@@ -17,37 +17,40 @@
  */
 package org.jboss.pnc.buildagent.common.security;
 
-import java.io.IOException;
 import java.io.UnsupportedEncodingException;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
-public class Md5 {
+public class Checksum {
 
-    private MessageDigest md;
+    private MessageDigest md5;
+    private MessageDigest sha512;
 
```

Comment:
Why the two checksums for the logs? The initial idea was to make sure the log is completely transferred, I picked MD5 which is faster than SHA, I was event thinking to use CRC.

Reply:
@thescouser89 ^

Reply:
Hey sure! If it's only used to very if the log is completely transferred, then md5 is good enough.

Woops @janinko  I've probably created more work than needed to you. Sorry!